README.md

```diff
@@ -29,41 +29,62 @@ programming quiz.
 Quick Start
 ===========
 
-#### Pre-Requisite
+#### Pre-Requisites
 
-1. Ensure [pip](https://pip.pypa.io/en/latest/installing.html) is installed
+1. Ensure you have Python available.
+1. Ensure [pip](https://pip.pypa.io/en/latest/installing.html) is installed.
 
 #### Installation
 
-1. Install the yaksh
+1. Install yaksh
     - For latest stable release
-    
+
             $ pip install yaksh
 
     - For the development version
 
             $ pip install git+https://github.com/FOSSEE/online_test.git
 
-1. In the terminal run
-        
-        yaksh create_demo [-p PATH] [project_name]
-    - ```project_name``` is the desired name of the django project.
-    - In case a ```project_name``` is not specified, the project is initialised with the name ```yaksh_demo```
-    - PATH is an optional flag to specify where the django project will be installed
-    - If PATH is not provided, the project is created in the current directory
+#### Short instructions
+
+To see a quick demo after installing yaksh do the following:
+
+    $ yaksh create_demo yaksh_demo
+    $ yaksh run yaksh_demo
+
+On another terminal start up the code server that executes the user code safely:
+
+    $ sudo yaksh run_code_server
```

Comment:
At the risk of stating the obvious,  yaksh has to be installed system wide as well. It would be better to explicitly mention this here.